code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -351,7 +334,11 @@
 	var/mob/living/carbon/human/H = M
 
 	// Prepare their return
-	prisoner_timer_handle = addtimer(CALLBACK(src, PROC_REF(handle_target_return), M, T), prison_time, TIMER_STOPPABLE)
+	if(isAntag(M))
+		prisoner_timer_handle = addtimer(CALLBACK(src, PROC_REF(handle_target_return), M, T), 10 MINUTES, TIMER_STOPPABLE)
```

Comment:
I'd prefer if 10 MINUTES was some kind of define here, this feels fairly arbitrary.

Reply:
made into its own var `felony_time`